elastic4s-handlers/src/main/scala/com/sksamuel/elastic4s/handlers/update/UpdateHandlers.scala

```diff
@@ -87,10 +87,9 @@ trait UpdateHandlers {
       request.scrollSize.foreach(params.put("scroll_size", _))
       request.waitForActiveShards.foreach(params.put("wait_for_active_shards", _))
       request.waitForCompletion.foreach(params.put("wait_for_completion", _))
-      request.slices.foreach {
-        case AutoSlices            => params.put("slices", "auto")
-        case NumericSlices(slices) => params.put("slices", slices)
-      }
+      request.slices.foreach(s =>
+        if (s == Slicing.AutoSlices) params.put("slices", Slicing.AutoSlicesValue) else params.put("slices", s)
```

Comment:
should this be 
```scala
 else params.put("slices", s.toString)
 ```
  as well?

Reply:
I'm not getting a compile error, so I think we're good there.